src/routes/index.tsx

```diff
@@ -23,4 +25,11 @@ export const router = createBrowserRouter([
     path: "/register",
     element: <RegistrationForm />,
   },
+  {path: '/create-post',
+    element: <CreatePost />,
+  },
+  {
+    path: '/posts',
+    element: <PostPage />,
+  },
```

Comment:
These routes belong in `children` array, so they are wrapped in the `ProtectedRoute` which will handle unauthorized requests to pages in our ui